execution/gethexec/executionengine.go

```diff
@@ -249,6 +252,34 @@ func (s *ExecutionEngine) EnablePrefetchBlock() {
 	s.prefetchBlock = true
 }
 
+func (s *ExecutionEngine) EnableMultigasCollector(config mgc.Config) error {
+	if s.mgcCollector != nil {
+		return nil
+	}
+
+	ch := make(chan *mgc.CollectorMessage, mgc.CollectorMsgQueueSize)
+	collector, err := mgc.NewCollector(config, ch)
+	if err != nil {
+		return err
+	}
+
+	arbos.SetMultiGasCollectorChan(ch)
+	s.mgcCollector = collector
+
+	return nil
+}
+
+func (s *ExecutionEngine) DisableMultigasCollector() {
```

Comment:
Is there anything that would actually call this method?

Reply:
No, i'll remove it